backend/apps/core/checks.py

```
@@ -0,0 +1,28 @@
+                    
+                    if journal_mode != "wal":
+                        errors.append(
+                            Error(
+                                f"SQLite database '{alias}' is not using WAL mode (current: {journal_mode}).",
+                                hint="Ensure your database settings include 'PRAGMA journal_mode=WAL;' in init_command.",
+                                obj=f"Database: {alias}",
+                                id="perf.E001",
+                            )
+                        )
+            except Exception as e:
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)